document/src/main/resources/META-INF/spring.factories

```diff
@@ -1,6 +1,6 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
   com.ritense.document.autoconfigure.DocumentAutoConfiguration, \
   com.ritense.document.autoconfigure.DocumentSnapshotAutoConfiguration, \
-  com.ritense.document.autoconfigure.SecurityAutoConfiguration, \
-  com.ritense.document.autoconfigure.LiquibaseAutoConfiguration, \
-  com.ritense.document.autoconfigure.RetryAutoConfiguration
+  com.ritense.document.autoconfigure.DocumentSecurityAutoConfiguration, \
```

Comment:
I'm not going to mention this for every autoconfiguration, but why rename these? They're already in the document module. It doesn't hurt or anything, but I don't see the benefit either. It's just inconsistent with the rest.

Reply:
We have the same class names in every module so we want to avoid which configuration we are using